.github/workflows/test.yml

```diff
@@ -111,10 +111,29 @@ jobs:
         env:
           PACTFLOW_PACT_FOUNDATION_TOKEN: ${{ secrets.PACTFLOW_PACT_FOUNDATION_TOKEN }}
 
+  pact-v2-verify:
+    if: "!contains(github.event.head_commit.message, '[ci-skip]')"
```

Comment:
This isn't needed. It looks like it's a built-in feature by GitHub: https://docs.github.com/en/actions/how-tos/manage-workflow-runs/skip-workflow-runs

Reply:
removed, ta